requirements.txt

```diff
@@ -1,4 +1,5 @@
 torch>=2.1.0
-fire
-pyarrow
-transformers
+fire==0.5.0
+pyarrow==15.0.0
+transformers==4.37.2
+ibm-fms>=0.0.3
```

Comment:
we might end up wanting to change this to be git/main too, but fine for now.

Reply:
In the `mypy` job it installs the version from main on top: `pip install ibm-fms@git+https://github.com/foundation-model-stack/foundation-model-stack@main`, but if the code will also depend on unreleased versions of fms we need to switch it in the main `requirements.txt`.

Reply:
yup, we can figure this out later. we may want tagged versions that depend on a specific fms version for reproducibility, but then also during development we're usually going to want to be using `main` not the specific fms version.